.travis.yml

```diff
@@ -3,7 +3,7 @@ language: python
 env:
   - ES_VERSION="7.7.1"
 python:
-  - "3.5"
+  - "3.6"
   - "3.7"
```

Comment:
Could we go up higher? (3.8 or even 3.9)

Reply:
I checked and am waiting for this:
https://github.com/NCI-GDC/esbuild/blob/develop/.travis.yml